### DISCUSSION TITLE
不懂就问，如何关闭脚本！

### DISCUSSION THREAD

**Discussion by leoicye** [the1812/Bilibili-Evolved#2546] [提问 / Q&A] [answered]
我找了一次又一次，可就是没有看见卸载脚本或关闭脚本的地方。其实倒也没什么，我想用B站自己的快捷键，可是我用不了，各种冲突，我很难受，我用不到这么多的功能，我单纯想请教一下“为什么外面没有关闭”的选项呀！我在油猴的管理面板里面也找不到“B站增强”什么的脚本，我把我安装的脚本都禁了，可它还是能够运行，为什么？？？怎么关呀！

**Comment by the1812** [comment, marked as answer]
你都不知道这脚本叫什么是怎么装上的...
![image](https://user-images.githubusercontent.com/26504152/142765823-beb87405-5fae-4aaa-82f3-4e4e282884ef.png)

**Reply by leoicye** [reply]
我当时就想着装一下玩一玩看一看的，之前在油猴的管理面板里面，我也不知道为什么，就是怎么都看不到这个脚本也没有显示在油猴的面板里面，今天又进入控制面板，终于找到它了，终于删掉了。谢谢！gethub弄的东西小但真的都是蛮厉害的。

**Comment by Geeyun-JY3** [comment]
脚本自己也有设置面板可以关功能的……

**Reply by leoicye** [reply]
主要是当时我左击油猴图标，以及在油猴的管理面板里面怎么都找不到那个脚本的位置，就慌了。我记得当时，我将我电脑所有的脚本以及插件都停止运行了，可它还是在，界面刷新了也还在，就懵逼了。而且我也忘记了它的脚本名字，以及找不到它的位置。但今天再次打开的时候，在油猴面板里看见了这个脚本，也终于是弄掉了。
